packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -6,8 +6,6 @@
     --mafs-bg: transparent;
     --mafs-fg: rgb(33, 36, 44); /* WB color.offBlack */
 
-    --mafs-line-color: rgba(33, 36, 44, 0.16); /* WB color.offBlack16*/
```

Comment:
@nishasy and I looked at this-- this variable is used for gridlines and needs to be offBlack16 or offBlack24

Reply:
@nedredmond I ended up changing it back to offBlack16, because I looked and offBlack24 is not there in WB. 24% was only added for certain colors, not black. My mistake!